public/lib/helpers/timeline_helper.js

```diff
@@ -49,16 +49,12 @@ define(function (require) {
      * @returns the label as a string
      */
     TimelineHelper.prototype.pluckLabel = function (hit, params, notify) {
-      let field;
-      if (params.labelFieldSequence) { // in kibi, we have the path property of a field
-        field = kibiUtils.getValuesAtPath(hit._source, params.labelFieldSequence);
-      } else {
-        field = _.get(hit._source, params.labelField);
-      }
-      if (field && (!_.isArray(field) || field.length)) {
-        return field;
-      }
-      return 'N/A';
+      // in kibi, we have the path property of a field
+      const field = params.labelFieldSequence ? params.labelFieldSequence[0] : params.labelField;
```

Comment:
Because `params.labelFieldSequence` is an array with one element - field name. But maybe I can just return the array, I'll check it